exp-player/addon/components/exp-video.js

```diff
@@ -11,6 +13,31 @@ export default ExpFrameBaseComponent.extend(FullScreen, MediaReload, {
     displayFullscreen: true,  // force fullscreen for all uses of this component, always
     fullScreenElementId: 'player-video',
 
+    spaceHandler: null,
+    didRender() {
```

Comment:
`didInsertElement` will only run once. This has the potential to run multiple times.